apps/dashboard/src/@/components/ui/tooltip.tsx

```diff
@@ -31,14 +31,20 @@ export { Tooltip, TooltipTrigger, TooltipContent, TooltipProvider };
 
 export function ToolTipLabel(props: {
   children: React.ReactNode;
-  label: string;
+  label: string | undefined;
   rightIcon?: React.ReactNode;
   leftIcon?: React.ReactNode;
 }) {
+  if (!props.label) {
+    return <>{props.children}</>;
```

Comment:
don't think you need the fragment here? Newer react types handle just returning props.children fine I believe?

Reply:
removed